frontend/public/components/namespace.jsx

```diff
@@ -139,111 +140,102 @@ const namespaceColumnInfo = Object.freeze({
   name: {
     classes: '',
     id: 'name',
-    title: 'Name',
   },
   displayName: {
     classes: 'co-break-word',
     id: 'displayName',
-    title: 'Display Name',
   },
   status: {
     classes: '',
     id: 'status',
-    title: 'Status',
   },
   requester: {
     classes: '',
     id: 'requester',
-    title: 'Requester',
   },
   memory: {
     classes: '',
     id: 'memory',
-    title: 'Memory',
   },
   cpu: {
     classes: '',
     id: 'cpu',
-    title: 'CPU',
   },
   created: {
     classes: '',
     id: 'created',
-    title: 'Created',
   },
   description: {
     classes: '',
     id: 'description',
-    title: 'Description',
   },
   labels: {
     classes: '',
     id: 'labels',
-    title: 'Labels',
   },
 });
 
-const NamespacesTableHeader = () => {
+const NamespacesTableHeader = (componentProps, t) => {
   return [
     {
-      title: namespaceColumnInfo.name.title,
+      title: t && t('namespace~Name'),
```

Comment:
Is `t` ever undefined? I don't think we should need these checks.

Reply:
@spadgett it could be undefined. When using col management, we sometimes only need only the metadata, and not the title strings.